lib/pluginmanager/remove.rb

```diff
@@ -21,21 +21,10 @@ def execute
 
     # since we previously did a gemfile.find(plugin) there is no reason why
     # remove would not work (return nil) here
-    if gemfile.remove(plugin)
-      gemfile.save
-
-      puts("Removing #{plugin}")
-
-      # any errors will be logged to $stderr by invoke!
-      # output, exception = LogStash::Bundler.invoke!(:install => true, :clean => true)
-      output = LogStash::Bundler.invoke!(:install => true, :clean => true)
+    exit(1) unless !::Bundler::LogstashUninstall.uninstall!(plugin)
```

Comment:
should this be `exit(1) unless ::Bundler::LogstashUninstall.uninstall!(plugin)`?
right now it's the same as `exit(1) if ::Bundler::LogstashUninstall.uninstall!(plugin)`